### DISCUSSION TITLE
Where to find documentation about components

### DISCUSSION THREAD

**Discussion by svenja-haas** [JedWatson/react-select#5889] [Q&A]
On the [documentation website](https://react-select.com/components) I can only ever find this list of all the components, but no descriptions of what which component is or does. This is very frustrating and confusing, especially if I want to do custom components where I need to know which one I should replace. Is there any documentation which explains the components or shows examples?

**Comment by manjushsh** [comment]


**Reply by svenja-haas** [reply]
this is not my question. my question is if there is any documentation of the existing components, not how to create a custom components.
